lib/response.js

```diff
@@ -368,6 +369,10 @@ res.sendFile = function sendFile(path, options, callback) {
     done = options;
     opts = {};
   }
+  
+  if(! ('etag' in opts) ) {
```

Comment:
Pleas do not use in for option detection. This makes the interface really confusing because you have to ensure that whatever you are passing in does not have that key anywhere in the prototype chain.